React/Base/RCTPlatform.m

```diff
@@ -39,7 +39,11 @@ @implementation RCTPlatform
     @"forceTouchAvailable": @(RCTForceTouchAvailable()),
     @"osVersion": [device systemVersion],
     @"systemName": [device systemName],
+#if TARGET_OS_TV
+    @"interfaceIdiom": interfaceIdiom(UIUserInterfaceIdiomTV),
+#else
     @"interfaceIdiom": interfaceIdiom([device userInterfaceIdiom]),
```

Comment:
I tested this manually and I don't understand why you need to TARGET_OS_TV, `[device userInterfaceIdiom]` should already return the right thing.